llm/server.go

```diff
@@ -921,6 +941,31 @@ func (s *llmServer) EstimatedVRAM() uint64 {
 	return s.estimatedVRAM
 }
 
+func (s *llmServer) GetGpuUsage() string {
```

Comment:
GibiBytes (format.HumanBytes2) might be appropriate since both available VRAM and usage is reported in MiB in logs

Reply:
Seems a bit less friendly
```
gpu usage:            1 GPU (33/33 layers) 5033.0 MiB
```

```
gpu usage:            1 GPU (27/33 layers) 26042.6 MiB (22738.0 MiB GPU)
```

Reply:
I'll add GibiBytes handling to HumanBytes2